knox-homepage-ui/homepage/assets/vkbeautify.js

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
I'd personally love to get rid of this file both here and in gateway-admin-ui. Either its something we can pull in via npm or we should have it minified. Right now its weird we straight copy this around.

Reply:
Done in both projects (pulled in by installing with npm)